src/jdk.jartool/share/classes/sun/tools/jar/resources/jar.properties

```diff
@@ -23,6 +23,10 @@
 # questions.
 #
 
+## tool
+
+jar.description=create an archive for classes and resources, and manipulate or restore individual classes or resources from an archive
```

Comment:
Not sure I love the use of  _manipulate_ but I understand that this is the wording from the tools guide for jar.